platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/IconFactory.java

```diff
@@ -24,11 +24,14 @@
 
 /**
  * Factory for creating {@link Icon} objects.
+ *
  * @see Icon
  */
 public final class IconFactory {
 
     private static final String ICON_ID_PREFIX = "com.mapbox.icons.icon_";
+    public static final Bitmap ICON_MARKERVIEW_BITMAP = Bitmap.createBitmap(1, 1, Bitmap.Config.ARGB_8888);
```

Comment:
I imagine so, but I can't find it in the docs. Is by default the result of `Bitmap.createBitmap(1, 1, Bitmap.Config.ARGB_8888)` transparent?